native/src/routes/__tests__/Landing.spec.tsx

```diff
@@ -73,6 +75,13 @@ describe('Landing', () => {
     expect(queryByText('Oldtown')).toBeFalsy()
   })
 
+  it('should show footer', () => {
+    mockCheckLocationPermission.mockImplementationOnce(async () => RESULTS.BLOCKED)
+    const { getByText } = renderLanding()
+    expect(getByText('cityNotFound')).toBeTruthy()
+    expect(getByText('clickHere')).toBeTruthy()
```

Comment:
Perhaps you could add a check that `navgiateToCityNotCooperating` is called on click?